meta-resin-common/recipes-devtools/mtools/mtools/initialize-direntry.patch

```diff
@@ -0,0 +1,11 @@
+diff -rup mtools-4.0.18.orig/direntry.c mtools-4.0.18/direntry.c
```

Comment:
We add a signed-off-by and upstream-status in patches, like so: https://github.com/resin-os/meta-resin/pull/675/files#diff-5adf707114efba588ffc2ecddb4f449bR17

Reply:
And a description please.